gridstatus/tests/test_ercot.py

```diff
@@ -32,6 +32,14 @@ def test_get_as_prices(self):
         assert df.columns.tolist() == as_cols
         assert df["Time"].unique()[0].date() == date
 
+        date = pd.Timestamp(2022, 11, 8).date()
```

Comment:
i would add a test that requests just a single day over a year ago. for example, i believe this currently fails


```python
self.iso.get_as_prices(date=today-pd.Timedelta(days=365))
```